syntax = "proto2";
package goby.middleware.protobuf;

message InterVehicleTransporterConfig
{
    optional int32 ttl = 1;
    optional int32 value_base = 2;
}

message TransporterConfig
{
    // if the publisher is also subscribed, should it receive a copy?
    // TODO: implement at the interprocess layer
    optional bool echo = 1 [default = false];

    optional InterVehicleTransporterConfig intervehicle = 10;
}
